larndsim/config/config.yaml

```diff
@@ -45,6 +45,8 @@ module0:
     PIXEL_LAYOUT_ID: [0, 0, 1, 0] 
     RESPONSE:        [response_44_v2a_50ns.npy, response_38_v2b_50ns.npy]
     RESPONSE_ID:     [0, 0, 1, 0] 
+    PIXEL_THRESHOLDS_FILE: [/global/cfs/cdirs/dune/www/data/2x2/simulation/larndsim_data/2x2_CRS_thresholds/thresholds_module0.npz, /global/cfs/cdirs/dune/www/data/2x2/simulation/larndsim_data/2x2_CRS_thresholds/thresholds_module1.npz, /global/cfs/cdirs/dune/www/data/2x2/simulation/larndsim_data/2x2_CRS_thresholds/thresholds_module2.npz, /global/cfs/cdirs/dune/www/data/2x2/simulation/larndsim_data/2x2_CRS_thresholds/thresholds_module3.npz]
```

Comment:
Hardcodes a dependence on NERSC filesystems. Are these files sufficiently small to include in the `bin` directory? Otherwise maybe we could include a script that downloads these from the nersc portal into the local bin directory?